03-task-manager/starter/quizAnswers2.txt

```diff
@@ -0,0 +1,32 @@
+
+1. 
+In this lesson, you created a middleware function called asyncWrapper. Why?
+
+It's a middleware to handle async error. 
+
+We have asynchronous operation and it becomes redundant for every controller setting up the bloc 'try{} catch(error)', 
+so a better solution is to use a middleware asyncWrapper to prevent the  repetition of code. 
+This way as well, it's easier to maintain the code and to handle different types of error
```

Comment:
Easier to maintain because you have a centralized file (or set of files) to consult for error handling behavior. 

Good job on both questions!

Reply:
Thank you so much !